shared/wallets/transaction/index.js

```diff
@@ -108,18 +109,39 @@ const Detail = (props: DetailProps) => {
   // u2026 is an ellipsis
   const textSentenceEnd = props.detailView && props.pending ? '\u2026' : '.'
 
-  const amount = props.isXLM ? (
-    <Text selectable={props.selectableText} type={textTypeExtrabold}>
-      {props.amountUser}
-    </Text>
-  ) : (
-    <React.Fragment>
-      Lumens worth{' '}
-      <Text selectable={true} type={textTypeExtrabold}>
-        {props.amountUser}
-      </Text>
-    </React.Fragment>
-  )
+  let amount
+  if (props.issuerDescription) {
+    // non-native asset
+    amount = (
+      <React.Fragment>
+        <Text selectable={props.selectableText} type={textTypeExtrabold}>
+          {props.amountUser}
+        </Text>{' '}
```

Comment:
Probably best to wrap these in a `Text` instead of `Fragment`,  the unwrapped spaces will crash mobile if `amount` is used in a different context.

Reply:
Also `amount` is a local variable in this function, so it shouldn't leak outside, and always ends up inside of a `Text`

I tried changing it to `Text` but then it expects me to make a decision about `type` (this prop seems required for Text) and I didn't know which one to use.